recipe/meta.yaml

```diff
@@ -19,18 +19,24 @@ build:
 
 requirements:
   host:
-    - python
+    - python >=3.6
     - pip
   run:
-    - python
+    - python >=3.6
     - pyyaml
     - click
+    - pandoc >=2.11.0.4
```

Comment:
Is pandoc a required dependence or is the purpose of this pin to make sure a compatible version of pandoc is in the environment should one be there?

The difference is whether or not you want `conda install panflute` to also install pandoc. If you do not require pandoc to run panflute, then a better option would be to use the run_constrained section (https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#run-constrained).

Reply:
@sergiocorreia, what are your thoughts on this?

Basically @beckermr is asking if we want pandoc as an optional constraint or mandatory. I'd recommend having it mandatory as the whole point of this PR and sergiocorreia/panflute#153 is to ensure the correct version of pandoc comes with `conda install panflute`. If it is only optional, and people only run `conda install panflute`, whatever other pandoc in their PATH would be used in panflute and causes problems. I think `pip install panflute` give people the freedom they want if they really want to manage pandoc separately.

@beckermr, I intend it to be mandatory. But may be asking @sergiocorreia's thought on this. Panflute while can be pure Python can don't call pandoc, using panflute not calling pandoc is not really a practical scenario. c.f. conda-forge/staged-recipes#13455. The original thought was to have 2 recipe, one is panflute, one is pandoc-panflute, where the later mandates pandoc with the right version. However they don't seem to think it makes sense this way.